packages/waku/src/lib/handlers/handler-dev.ts

```diff
@@ -228,9 +228,15 @@ export function createHandler<
         item.url !== viteUrl &&
         !item.url.includes('?html-proxy')
       ) {
+        const { code } = (await vite.transformRequest(item.url))!;
```

Comment:
I personally didn't like my hack from L218, and this makes me concerned more.
Let's look for a better solution (I'll open a new issue), and merge this as a temporary workaround for now.